src/Mutator/Arithmetic/Assignment.php

```diff
@@ -59,7 +59,11 @@ public static function getDefinition(): ?Definition
 TXT
             ,
             MutatorCategory::SEMANTIC_REDUCTION,
-            null
+            null,
+            <<<'DIFF'
+- $a += $b;
++ $a = $b;
+DIFF
```

Comment:
You can now do:

```php
            <<<'DIFF'
            - $a += $b;
            + $a = $b;
            DIFF
```

Reply:
I would prefer to run something like Rector to change it in the whole project, if needed, and if this possible ;) but for now leave as is if you don't mind

Reply:
fair